pkg/services/sqlstore/sqlstore.go

```diff
@@ -460,13 +460,23 @@ var testSQLStoreMutex sync.Mutex
 type InitTestDBOpt struct {
 	// EnsureDefaultOrgAndUser flags whether to ensure that default org and user exist.
 	EnsureDefaultOrgAndUser bool
+	AccessControlEnabled    bool
```

Comment:
Would be nicer if this was a list of feature toggles to enable to make it work for any feature toggle. Then might not need to override `isFeatureToggleEnabledForTest` - it can check this list and the predefined one (or remove the predefined one in favor of this). WDYT?